src/profile-logic/call-tree.js

```diff
@@ -81,7 +81,7 @@ export class CallTree {
     this._rootTotalTime = rootTotalTime;
     this._rootCount = rootCount;
     this._displayDataByIndex = new Map();
-    this._children = new Map();
+    this._children = new Array(callNodeTable.length);
```

Comment:
I'd set it to `[]` here and reinstanciate it in preloadChildrenCache with the full length. The goal is to not preallocate a lot of memory if we don't need it (which is the case for the call tree).